Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -40,6 +31,19 @@ class SignInViewController: UIViewController, Validation {
                           alertActionTitle: "Try again" ,
                           alertDelegate: self)
     }
+    // MARK: - Vars/Lets
```

Comment:
Recommended order for this class 😄 

@IBOutlet
Vars/Lets
Life Cycle
@IBAction
Functions

Reply:
Noted. Thank you for the advice👍🏻